### ISSUE TITLE
"Invalid data" repeatedly announced when inputting answers

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#7899]
## Context

From client: results of their form in fable testing:

> Ran 20 accessibility tests using Fable (10 in English and 10 in French). The accessibility tests mainly focused on screen reader compatibility, with a couple tests using screen magnification tools.

https://cds-snc.freshdesk.com/a/tickets/30088
- “Invalid data” being repeatedly announced while navigating options or typing in fields
- This can be confusing and frustrating for screen-reader users.

> "When I type in the form field, Voiceover keeps saying invalid data as I type each character, which is very annoying. To select the language options, Voiceover also only read invalid data if I tried to move by next element, I had to explore the screen by touch to find the options."

## Findings 

**4 issues related to “invalid data” being announced as users navigate through form options or type in form fields** 

(testers used VoiceOver with Safari or NVDA with Firefox or NVDA with Chrome)

- “The error is not announced once the box is checked, but returns if a box is again set to be unchecked. This would suggest to me, wrongly, that all the boxes simply have to be checked.”
- “When I type in the form field, Voiceover keeps saying invalid data as I type each character, which is very annoying.”
